### DISCUSSION TITLE
[Entrega 1] Consultas del dominio

### DISCUSSION THREAD

**Discussion by RaniAgus** [dds-jv/foro#221] [TPA - 2021] [answered]
¡Buenas! Les traemos par de consultas cortitas:

1. Al verificar los requerimientos de una mascota perdida hay un atributo que nos llamó la atención:
> Lugar donde la encontró, el cual será seleccionable a través de un mapa

¿Esto requiere importar una library que incluya geolocalización o por "mapa" se refiere a una ubicación en 2D (X,Y)? 

2. En cuanto a los datos de contacto: 
> Al menos un dato de contacto. Un contacto debe contener, mínimamente:
> - Nombre y Apellido
> - Teléfono
> - Email

¿Todos los contactos deben ser de la misma persona (ya sea dueña o rescatista) o puede incluir también a un familiar/amigo, por ejemplo? De ser la segunda, ¿se debe verificar que ese "al menos un dato de contacto" sea de la misma persona o existe la posibilidad de agregar solo familiares/amigos?

¡Saludos!

**Comment by roli93** [comment, marked as answer]
Hola!

1) Ambas opciones pueden ser válidas, tanto hacer un modelado propio como utilizar aquel provisto por una biblioteca de terceros.
2) Los datos de contacto son de la misma persona involucrada.

De todas formas, tengan en cuenta este tipo de consultas les serán mejor respondidas por su ayudante designado, que va a conocer mejor las particularidades de cada grupo y TP (en especial de cara a entregas futuras)

Saludos!

**Reply by RaniAgus** [reply]
> De todas formas, tengan en cuenta este tipo de consultas les serán mejor respondidas por su ayudante designado, que va a conocer mejor las particularidades de cada grupo y TP (en especial de cara a entregas futuras)

Buenísimo, lo tendremos en cuenta para la próxima. ¡Gracias!
